.github/workflows/coverage.yml

```
@@ -0,0 +1,32 @@
+name: coverage
+
+on:
+  push:
+    branches:
+      main
+  pull_request:
+    branches:
+      main
+
+jobs:
+  build-with-regex:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+      - name: Install
+        run: |
+          pip install regex
+          python setup.py install
+      - name: Tests
+        run: |
+          pip install nose coveralls
+          coverage run --source=titlecase setup.py nosetests
+
+      - name: Coveralls
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.COVERALLS_TOKEN }}
```

```suggestion
          github-token: ${{ secrets.github_token }}
```

Looks like this workflow actually wants the github token, not the coveralls token?